### ISSUE TITLE
cube-builder no python2 required by wrlinux-x

### ISSUE THREAD

**Issue by outbackdingo** [OverC/meta-overc#289]
curious if cube-builder was based around wrlinux-x .. which requires python2, and python3 is installed

so i guess all references need to be changed for it to functionally work

./wrlinux-x/setup.sh --all-layers --mirror
which: no python2 in (/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/sbin:/sbin)
WRLinux setup requires 'python2'.
Please install python2.

**Comment by zeddii** [comment]
The initial package list for builder was more based on what you'd find in a headless debian/ubuntu install, but yes, it was targeted to be self hosting. That being said, we only did that testing using the Yocto/OE layers and not a wrlinux-x install.

We also did go through a process of purging python2 from the components, since having both 2 and 3 around was inflating the essential image size, etc.

The short story is. You'd need to add python2 yourself to the cube-builder package list, or get a wrlinux-x version that has moved on past needing python2.

Tagging @masselstine again. In case he has comments.
